lib/services/useracl.go

```diff
@@ -153,6 +155,9 @@ func NewUserACL(user types.User, userRoles RoleSet, features proto.Features, des
 		billingAccess = newAccess(userRoles, ctx, types.KindBilling)
 	}
 
+	//TODO(jakule): move later to the cloud section.
```

Comment:
If long-term we're planing to provide self-hosted access graph, perhaps there's no need to move this to `cloud` section, only to move out later?